docs_user/modules/proc_adopting-compute-services-to-the-data-plane.adoc

```diff
@@ -126,6 +126,9 @@ endif::[]
 ifeval::["{build}" == "downstream"]
 $(cat <path_to_SSH_key> | base64 | sed \'s/^/        /')
 endif::[]
+ifeval::["{OpenStackPreviousInstaller}" == "director_operator"]
+      $(oc exec -n ${OSPDO_NAMESPACE} -t openstackclient openstackclient -- cat /home/cloud-admin/.ssh/id_rsa | base64 | sed 's/^/        /')
```

Comment:
```suggestion
      $(oc exec -n $<ospdo_namespace> -t openstackclient openstackclient -- cat /home/cloud-admin/.ssh/id_rsa | base64 | sed 's/^/        /')
```
Customers will need to replace <ospdo_namespace> with their own namespace, right?

Reply:
right